docs/usage/setup.md

```diff
@@ -130,6 +130,8 @@ Enable this to use Z-Wave JS UI as only a Control Panel
      - `property`: the value [property](https://zwave-js.github.io/node-zwave-js/#/api/valueid)
      - `propertyKey`: the value [propertyKey](https://zwave-js.github.io/node-zwave-js/#/api/valueid)
 
+     Note that `<mqtt_name>` will also be the numerical id of if the controller node
```

Comment:
This is not correct, mqtt name will be the `name` setting set in MQTT settings

Reply:
@spacemanspiff2007 Are you sure you don't have a `MQTT_NAME` env var set to `1`?

The code is pretty clear: 

https://github.com/zwave-js/zwave-js-ui/blob/0e6251a06dc9ff077c1849dacd0d5ba87852abc1/api/lib/MqttClient.ts#L346-L347

Where prefix is:

https://github.com/zwave-js/zwave-js-ui/blob/0e6251a06dc9ff077c1849dacd0d5ba87852abc1/api/lib/MqttClient.ts#L78-L79

Reply:
I use a docker compose file and I am 100% sure that ``MQTT_NAME`` is not set there.
Maybe  ``MQTT_NAME`` is set to `1` by default?

I run an older version (9.24.0), maybe something changed?
I'll try updating to see if the issue still persists ...

Reply:
> No the less I think the docs would benefit if they show where <mqtt_name> comes from.

I agree with that but what it's wrote there is wrong at least all my instances are not working like that so I'm wondering why you have that `1` there...

Reply:
Hmm - as you can see in the above screenshot it indeed sends a success message so something is happening.

![grafik](https://github.com/user-attachments/assets/d4714981-d695-4c4f-b93b-9ef55b77146c)

Reply:
> so I'm wondering why you have that `1` there

That's the topic I publish to?

Reply:
> Note that I copy pasted the wrong url to the docs above. The correct one would be [the zwave-apis](https://zwave-js.github.io/zwave-js-ui//#/guide/mqtt?id=z-wave-apis).

It doesn't matter, the mqtt-name is always that. Anyway could you try to connect a shell to container and run `echo $MQTT_NAME` ?

Reply:
![grafik](https://github.com/user-attachments/assets/fbf0d7cf-b493-4a47-a31a-6fe58e769f62)

Reply:
Ok so the publish now is ok, about the response to that topic I may know what's the reason, I have to make the check on name more strict (I only check the prefix but since I changed the way I do subscribes it may be that I also get others)

Reply:
See https://github.com/zwave-js/zwave-js-ui/commit/0e08e3a79e02656f47dc5b058ec08c02bbf6d90f